modules/compute/gke-node-pool/outputs.tf

```diff
@@ -14,25 +14,26 @@
   * limitations under the License.
   */
 
-output "node_pool_name" {
+output "node_pool_names" {
   description = "Name of the node pool."
-  value       = google_container_node_pool.node_pool.name
+  value       = google_container_node_pool.node_pool[*].name
 }
 
 locals {
   is_a_series = local.machine_family == "a2"
   last_digit  = trimsuffix(try(local.machine_vals[2], 0), "g")
 
   # Shared core machines only have 1 cpu allocatable, even if they have 2 cpu capacity
-  vcpu        = local.machine_shared_core ? 1 : local.is_a_series ? local.last_digit * 12 : local.last_digit
-  useable_cpu = local.set_threads_per_core ? local.threads_per_core * local.vcpu / 2 : local.vcpu
+  # Logic updated to work with TPUs
```

Comment:
Can you add a more meninful comment explaining why these numbers go to 0 for vcpu and `usable_cpu`?

Reply:
Good catch! I forgot to add a note here.
This is something to fix in the next iteration. The idea is to try separating out this piece so that the PR is not too big.

Reply:
Please make sure it works for CPU only machines.., which where the main target of this code I think.